### DISCUSSION TITLE
Google Play OBF app version

### DISCUSSION THREAD

**Discussion by sashabrava** [openfoodfacts/openfoodfacts-androidapp#3964] [Q&A]
Google Play Market still offers version 2.7.0 of Open Beauty Facts application, and latest comment there says app wasn't working well. Update on current 3.2.9 version would definitely improve app rating there.

Additionally OBF app (fresh version, compiled from source code) tries to load page "https://ssl-api.openbeautyfacts.org/data/invalid-barcodes.json" on launch, but its code is 404 and app shows wrong Toast about missing Internet connection (however, adding items to database works). App expects to see JSON there, maybe uploading an empty one (or the one from OpenFoodFacts from similar link) on the server could fix the issue.
![Screenshot_2020-04-14-22-45-49-682_openfoodfacts github scrachx openbeauty](https://user-images.githubusercontent.com/11160215/79272629-001e2a00-7ea2-11ea-9418-39d951c4a159.png)

**Comment by teolemon** [comment]
This should be fixed (the invalid-barcodes for OBF, OPF and OPFF).
The reason we're not releasing the new version of Open Beauty Facts is that it does not have the ingredient detection of V1.
Once we have it, it would be a snap.

**Comment by teolemon** [comment]
I'm talking about this feature:
![image](https://user-images.githubusercontent.com/1689815/79306304-451f7c00-7ef5-11ea-9f3e-e9ad702d87c6.png)

We don't have any Beauty Score (no scientist has created one so farà, so if we want to pretend to have an appeal beyond hard-core opendata people, we need to have it. It's a list of 17 flagged substances by the major French consumer union.

**Comment by sashabrava** [comment]
@teolemon If you're searching info about potentially dangerous chemicals among ingredients, maybe you'll be interested to read about [California Safe Cosmetics Program](https://www.cdph.ca.gov/Programs/CCDPHP/DEODC/OHB/CSCP/Pages/CSCP.aspx). They have public researches about chemicals, and the list is bigger than the one you mentioned.
